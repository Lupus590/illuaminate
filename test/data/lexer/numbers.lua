0x2
2
.2
2.
2.e2
2...3
