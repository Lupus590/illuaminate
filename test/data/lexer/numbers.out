0x2
(Node.Whitespace "\n")
2
(Node.Whitespace "\n")
.2
(Node.Whitespace "\n")
2.
(Node.Whitespace "\n")
2.e2
(Node.Whitespace "\n")
2...3
(Node.Whitespace "\n")
end of file
